gobblin-runtime/src/main/java/org/apache/gobblin/runtime/spec_store/MysqlBaseSpecStore.java

```diff
@@ -72,6 +73,7 @@ public class MysqlBaseSpecStore extends InstrumentedSpecStore {
 
   public static final String CONFIG_PREFIX = "mysqlBaseSpecStore";
   public static final String DEFAULT_TAG_VALUE = "";
+  public static final String modificationTimeKey = "modified_time";
```

Comment:
Can we make this a Configuration Key or a key associated with the FlowSpec rather than the SpecStore? Since the property is part of the flowSpec it will encapsulate better